### ISSUE TITLE
How to type `C-c m`?

### ISSUE THREAD

**Issue by salutis** [susam/devil#1]
I have tried `<EVIL> c m` but it does not work.  Ideas?

P.S. `;` is **the** choice for `<EVIL>` and should be the _default_.

**Comment by susam** [comment]
Hi @salutis,

Thank you for creating this issue. For typing something like `C-c m`, I use a configuration like this:

```elisp
(setq devil-translations
      '(("%k z" . "C-")
        ("%k %k" . "%k")
        ("%k m m" . "M-")
        ("m z" . "M-")
        ("m m" . "m")
        ("%k" . "C-")
        ("m" . "M-")))
```

Assuming the `devil-key` is `;` we now can type `; c m m` to get `C-c m`. This configuration follows the existing general trend of double-typing the special character to get a single special character. Another trend this follows is using a `z` in between to prevent double-typed special character from becoming a single special character. Here are some translation examples with this configuration:

* `; c m m` => `C-c m`
* `; c m z m` => `C-c M-m`
* `; c m z z` => `C-c M-z`

I have to admit though that I don't use the contrived `z`-based key sequences often. But they are there in case one needs them. I hope this helps. I might include the additional translations for `m z` and `m m` presented above in the default translation rules in a future release. I'll update this issue and post a comment here once that is done.

**Comment by susam** [comment]
The additional translation rules mentioned in my previous comment are now included in this package by default. They were added in commit fc62e58 and available in this package since version 0.4.0.
